src/t_stream.c

```diff
@@ -2804,7 +2804,7 @@ void xsetidCommand(client *c) {
     notifyKeyspaceEvent(NOTIFY_STREAM,"xsetid",c->argv[1],c->db->id);
 }
 
-/* XACK <key> <group> <id> <id> ... <id>
+/* XACK <key> <group> <id> [<id>...]
```

Comment:
i don't think we need to make an effort to match the formal argument convention rather than an example.
no one parses these, it's just to let the reader understand the syntax.

Reply:
roll back the updated comments, Thanks